src/router/Router.js

```diff
@@ -4,15 +4,17 @@ import {
     createBrowserRouter,
     createRoutesFromElements,
 } from 'react-router-dom';
-import { AuthLayout } from '../layouts';
+import { AuthLayout, MainLayout } from '../layouts';
+import { paths } from '../shared';
 import { LoginView } from '../views';
 
 export const Router = createBrowserRouter(
     createRoutesFromElements(
         <Route path='/'>
             <Route path='/' element={<AuthLayout />}>
-                <Route path='login' element={<LoginView />} />
+                <Route path={paths.LOGIN} element={<LoginView />} />
```

Comment:
Just a suggestion - instead of importing everything from everywhere (I mean layouts and views by that) you can create AuthRouter and DashboardRouter and import them from layouts folder. And there you could define any layout specific routes.

So:
root router - LayoutRouters
layout router - respective views routes